### ISSUE TITLE
Transparent Application is a Black Screen 

### ISSUE THREAD

**Issue by sundaram123krishnan** [waycrate/exwlshelleventloop#50]
So, in the counter example i had tried implementing styles for the application.

```rust
fn style(&self) -> theme::Application {
        theme::Application::custom(|theme: &Theme| iced::application::Appearance {
            background_color: iced::Color::TRANSPARENT,
            text_color: iced::Color::BLACK,
        })
    }
 ```
Instead, it displays a black screen
![transparent](https://github.com/user-attachments/assets/7ba8a115-bc86-4f44-96ba-10673afe213e)

**Comment by Decodetalkers** [comment]
Yes, the surface should be set as transparent first

**Comment by Decodetalkers** [comment]
I will solve it now

**Comment by Decodetalkers** [comment]
Emm.. failed.. seems hard..

**Comment by Decodetalkers** [comment]
https://github.com/waycrate/exwlshelleventloop/pull/51 I do not know why this happened... I think it should work..

**Comment by Decodetalkers** [comment]
![image](https://github.com/user-attachments/assets/08f9af4c-f1bf-4995-8798-dcbecc85bec0)

origin iced also cannot transparent.. is that because of the wgpu dependence is break?

**Comment by sundaram123krishnan** [comment]
Yes, when building from master branch, it works fine!

**Comment by sundaram123krishnan** [comment]
This issue is fixed  after updated `iced` to `0.13`.
can be closed?
@Decodetalkers

**Comment by Decodetalkers** [comment]
> This issue is fixed  after updated `iced` to `0.13`.
> can be closed?
> @Decodetalkers 

I think now it can be closed

**Comment by sundaram123krishnan** [comment]
yes
